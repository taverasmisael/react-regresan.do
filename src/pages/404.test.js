import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './404'

describe('Pages/404', () => {
  describe('rendering', () => {
    it('renders without crashing', () => {
      shallow(<NotFound />)
    })
    it('matches snapshot', () => {
      expect(shallow(<NotFound />)).toMatchSnapshot()
    })
  })
})
