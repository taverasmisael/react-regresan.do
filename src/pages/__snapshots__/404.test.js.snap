// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/404 rendering matches snapshot 1`] = `
<React.Fragment>
  <h1>
    Not Found
  </h1>
  <WithStyles(Button)
    color="primary"
    onClick={[Function]}
    raised={true}
  >
    Login
  </WithStyles(Button)>
</React.Fragment>
`;
